src/App.js

```diff
@@ -0,0 +1,24 @@
+import { Route, Routes } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/Books/ConfigureStore';
```

Comment:
- [ ] Kindly fix the import path, it should be `./redux/books/ConfigureStore` 👍

Reply:
ok fixed